doc/source/development/extending.rst

```diff
@@ -74,10 +74,11 @@ applies only to certain dtypes.
 Extension types
 ---------------
 
-.. warning::
+.. note::
 
-   The :class:`pandas.api.extensions.ExtensionDtype` and :class:`pandas.api.extensions.ExtensionArray` APIs are new and
-   experimental. They may change between versions without warning.
+   The :class:`pandas.api.extensions.ExtensionDtype` and :class:`pandas.api.extensions.ExtensionArray` APIs were
+   experimental prior to pandas 1.4. Starting with version 1.4, future changes will follow
```

Comment:
shouldn't we say prior to 1.5 (nbd deal really)

Reply:
OK.  Fixed in next commit.